File: Basic Coding Problems/01 Fundamentals/011 Write a program to find power of any number x ^ y/C/power.c
#include <stdio.h>
#include <math.h>

int main() {
    int base, power, result;

    printf("Enter the base: ");
    scanf("%d", &base);

    printf("Enter the power: ");
    scanf("%d", &power);

    result = pow(base, power);

    printf("The value of %d ^ %d is %d\n", base, power, result);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/011 Write a program to find power of any number x ^ y/Java/power.java
import java.util.Scanner;

public class power {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the base: ");
        int base = scanner.nextInt();

        System.out.print("Enter the power: ");
        int power = scanner.nextInt();

        int result = (int) Math.pow(base, power);

        System.out.println("The value of " + base + "^" + power + " is " + result);
    }
}